tensorflow_datasets/core/visualization/audio_visualizer.py

```diff
@@ -0,0 +1,75 @@
+""" Audio Visualization
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+
+from absl import logging
+
+from tensorflow_datasets.core import dataset_utils
+from tensorflow_datasets.core import features as features_lib
+from tensorflow_datasets.core import lazy_imports_lib
+from tensorflow_datasets.core.visualization import visualizer
+
+class AudioGridVisualizer(visualizer.Visualizer):
+  def match(self, ds_info):
+    audio_keys = visualizer.extract_keys(ds_info.features, features_lib.Audio)
+    return len(audio_keys) > 0
+
+  def show(
+      self,
+      ds_info,
+      ds,
+      rows=2,
+      cols=2,
+      plot_scale=3.,
+      image_key=None,
+  ):
+    """Display the dataset.
+
+      Args:
+        ds_info: `tfds.core.DatasetInfo` object of the dataset to visualize.
+        ds: `tf.data.Dataset`. The tf.data.Dataset object to visualize. Examples
+          should not be batched. Examples will be consumed in order until
+          (rows * cols) are read or the dataset is consumed.
+        rows: `int`, number of rows of the display grid.
+        cols: `int`, number of columns of the display grid.
+        plot_scale: `float`, controls the plot size of the images. Keep this
+          value around 3 to get a good plot. High and low values may cause
+          the labels to get overlapped.
+        image_key: `string`, name of the feature that contains the image. If not
+          set, the system will try to auto-detect it.
+      """
+    import random
```

Comment:
Random is a standard Python module, so installed in every users, so it is safe to keep it in the global scope